desafio-02/JonataMSilva/python/numeros_primos.py

```diff
@@ -0,0 +1,12 @@
+def encontrar_num_primo(numero):
```

Comment:
Agora que fez as alterações percebi, que essa função poderia ser melhor nomeada. Quanto mais descritivo for suas funções e seu código melhor sera entendido por vc ou por outros. No caso aqui o que a função esta fazendo vc passa um numero pra ela. E o que ela faz ?